terraform/etl_integration/main.tf

```diff
@@ -94,7 +94,7 @@ CREATE TABLE ${local.spanner_table} (
   passenger_count INT64,
```

Comment:
Remove the changes of this file from this PR. If this file needs to be updated (I think it needs, it seems to have formatting issues), file a PR only with these changes.